### PR TITLE
Release v0.35.1

### PR DESCRIPTION
**Pull request by rafalstepien**

### CODE REVIEW

File: peppy/project.py [hunk 1]
```diff
@@ -472,7 +412,7 @@ def load_samples(self):
             self[SUBSAMPLE_DF_KEY] = None
 
         for _, r in self[SAMPLE_DF_KEY].iterrows():
-            samples_list.append(Sample(r.dropna(), prj=self))
+            samples_list.append(Sample(r, prj=self))
```
Comment (Khoroshevskyi) [inline peppy/project.py:415]: @rafalstepien are you sure that here shouldn't be dropna?

Comment (rafalstepien) [reply peppy/project.py:415]: I'm sure :)

**Review by Khoroshevskyi — APPROVED** [review]

**Review by nleroy917 — APPROVED** [review]
LGTM

**Review by nsheff — APPROVED** [review]

**Comment by nsheff** [comment]
@rafalstepien ok go ahead and release. I just changed to 0.35.1 because by my estimate, this is a PATCH and not a MINOR release.

**Pull request merged**